coalib/core/Core.py

```diff
@@ -339,14 +339,13 @@ def run(bears, result_callback):
 
             def result_callback(result):
                 pass
+    :param executor:
+        Custom executor used to run the bears.
```

Comment:
(Especially that `ProcessPoolExecutor` will use as many processes as available cores)